### DISCUSSION TITLE
Is there any articles about the comparison of `nanostore` with other state management lib?

### DISCUSSION THREAD

**Discussion by sudoaugustin** [ai/nanostores#139] [Q&A]
After reading the docs, I want to jump from `recoil` to `nanostore` .  But I want to ensure I won't have to go back to `recoil` for performance or other reasons. I googled for comparison articles, but I didn't find any. 

So I'm stating the pros I know. Anyone can share any articles or their thoughts about `nanostore` in the thread.

### Pros
- Small Bundle Size
- Can update the state outside of `react-dom`.

**Comment by ai** [comment]
There is no such article right now.

The main benefits behind Nano Stores is not size, but the idea of moving business logic from React components to stores. Nano Stores is not just a storage for the data. The idea is that you will move validations, AJAX requests, etc into the stores.

It will allow to simplify tests and make all of your React components very simple.

**Comment by AjaxSolutions** [comment]
There are article comparing Recoil and Jotai, and Jotai usually scores higher.  Nano Stores looks even than Jotai IMHO.

**Comment by dyaa** [comment]
To me, `nanostore` is about separating the business logic from `react`, and having that business logic as framework agnostic since it supports many other frameworks other than react, which looks like not the same when it comes to `Jotai`.

**Reply by dyaa** [reply]
I see this issue is mainly the downside of using these types of libraries https://github.com/nanostores/nanostores/issues/80

**Comment by AjaxSolutions** [comment]
[Why Nano Stores?](https://docs.astro.build/en/core-concepts/sharing-state/#why-nano-stores)
